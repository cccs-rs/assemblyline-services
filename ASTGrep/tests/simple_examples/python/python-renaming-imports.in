from base64 import b64encode, something as invoke

import base64 as xyz

import torch.nn as nn
from torch.nn.parallel import DistributedDataParallel

from subprocess import run as run

import numpy as np

nn()
torch.nn()
xyz.something("aaa")

run("aaa")
np.array()

invoke("aaaa")

def aaa():
    invoke("aaaa")