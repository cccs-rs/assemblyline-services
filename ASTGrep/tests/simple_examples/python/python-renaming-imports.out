from base64 import b64encode, something as invoke

import base64 as xyz

import torch.nn as nn
from torch.nn.parallel import DistributedDataParallel

from subprocess import run as run

import numpy as np

torch.nn()
torch.nn()
base64.something("aaa")

run("aaa")
np.array()

something("aaaa")

def aaa():
    something("aaaa")